modin/pandas/test/test_groupby.py

```diff
@@ -1594,8 +1604,8 @@ def test_not_str_by(by, as_index):
     )
 
     modin_groupby_equals_pandas(md_grp, pd_grp)
-    df_equals(md_grp.sum(), pd_grp.sum())
-    df_equals(md_grp.size(), pd_grp.size())
-    df_equals(md_grp.agg(lambda df: df.mean()), pd_grp.agg(lambda df: df.mean()))
-    df_equals(md_grp.dtypes, pd_grp.dtypes)
-    df_equals(md_grp.first(), pd_grp.first())
+    eval_general(md_grp, pd_grp, lambda grp: grp.sum())
```

Comment:
some of the corner cases at this test are not supported by pandas since now and raise an exception, wrapped function execution to `eval_generate` to check exceptions concordance